src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -1556,4 +1577,15 @@ private static String logErrorMessage(ApiException apiEx) {
         ? apiExMsg
         : apiEx.getCode() + " " + apiEx.getMessage());
   }
+
+  public List<Property> buildProperties(Map<String, String> propertyMap) {
```

Comment:
optional Usually we use streams

return propertyMap.stream().map....
    .collect(toList());